tekton/tests/conftest.py

```diff
@@ -0,0 +1,121 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import time
+from contextlib import ExitStack
+
+import pytest
+
+from datadog_checks.dev import get_here, run_command
+from datadog_checks.dev.kind import kind_run
+from datadog_checks.dev.kube_port_forward import port_forward
+
+HERE = get_here()
+
+
+def setup_tekton():
+    run_command(["kubectl", "create", "namespace", "tekton-operator"])
+    run_command(["kubectl", "apply", "-f", os.path.join(HERE, "kind", "tekton-operator.yaml"), "-n", "tekton-operator"])
+    time.sleep(30)
+    run_command(
+        ["kubectl", "wait", "pods", "--all", "--for=condition=Ready", "--timeout=300s", "-n", "tekton-operator"]
+    )
+    time.sleep(60)
+    run_command(
+        [
+            "kubectl",
+            "wait",
+            "pods",
+            "-l",
+            "app=tekton-pipelines-controller",
+            "--for=condition=Ready",
+            "--timeout=300s",
+            "-n",
+            "tekton-pipelines",
+        ]
+    )
+    run_command(
+        [
+            "kubectl",
+            "wait",
+            "pods",
+            "-l",
+            "app=tekton-triggers-controller",
+            "--for=condition=Ready",
+            "--timeout=300s",
+            "-n",
+            "tekton-pipelines",
+        ]
+    )
+
+    run_command(
+        ["kubectl", "apply", "-f", os.path.join(HERE, "kind", "tekton-pipeline-hello.yaml"), "-n", "tekton-pipelines"]
+    )
+
+    run_command(
+        [
+            "kubectl",
+            "apply",
+            "-f",
+            os.path.join(HERE, "kind", "tekton-pipelinerun-hello.yaml"),
+            "-n",
+            "tekton-pipelines",
+        ]
+    )
+
+    for task in ("hello", "sleep"):
+        for kind in ("task", "pipeline"):
+            run_command(
+                [
+                    "kubectl",
+                    "apply",
+                    "-f",
+                    os.path.join(HERE, "kind", f"tekton-{kind}-{task}.yaml"),
+                    "-n",
+                    "tekton-pipelines",
+                ]
+            )
+
+            run_command(
+                [
+                    "kubectl",
+                    "apply",
+                    "-f",
+                    os.path.join(HERE, "kind", f"tekton-{kind}run-{task}.yaml"),
+                    "-n",
+                    "tekton-pipelines",
+                ]
+            )
+
+
+@pytest.fixture(scope='session')
+def dd_environment(dd_save_state):
```

Comment:
```suggestion
def dd_environment():
```
afaict you're not using that fixture

Reply:
Oh true I don't anymore!